newrelic_lambda_cli/integrations.py

```diff
@@ -634,8 +670,15 @@ def remove_license_key(input):
     click.echo(
         "Waiting for stack deletion to complete, this may take a minute... ", nl=False
     )
-    client.get_waiter("stack_delete_complete").wait(StackName=LICENSE_KEY_STACK_NAME)
-    success("Done")
+
+    try:
+        client.get_waiter("stack_delete_complete").wait(
+            StackName=LICENSE_KEY_STACK_NAME
+        )
+    except botocore.exceptions.WaiterError as e:
+        failure(e.last_response["Status"]["StatusReason"])
```

Comment:
Same control flow change here.

Reply:
This exception is not handled. So won't re-raise here.